docker/metadata.json

```diff
@@ -1117,6 +1117,22 @@
             "_schemaURL": "https://openlineage.io/spec/facets/1-0-0/DatasourceDatasetFacet.json",
             "name": "food_delivery_db",
             "uri": "postgres://food_delivery:food_delivery@postgres:5432/food_delivery"
+          },
+          "dataQualityAssertions": {
+            "_producer": "https://some.producer.com/version/1.0",
+            "_schemaURL": "https://openlineage.io/spec/facets/1-0-0/DataQualityAssertionsDatasetFacet.json",
+            "assertions": [
+              {
+                "assertion": "not_null",
```

Comment:
It's great that we are adding data quality checks! But, there's no reference to `user_name` in the [seed data](https://github.com/MarquezProject/marquez/blob/main/docker/metadata.json) for this dataset. Mind if we use columns that are defined in `public.delivery_7_days`?

Reply:
Make sense, I have added changes, I hope that it's what you expect

Reply:
Perfect!